hxd/Window.hl.hx

```diff
@@ -339,6 +357,7 @@ class Window {
 			46 => K.QWERTY_PERIOD,
 			47 => K.QWERTY_SLASH,
 			59 => K.QWERTY_SEMICOLON,
+			60 => K.QWERTY_BACKSLASH,

```

Comment:
This was unrelated to the issue, but fixes that you can detect the key above tab on Macs when using the english layout. 
![image](https://user-images.githubusercontent.com/494817/69582976-0dfa0e00-0fda-11ea-99b2-920db0eb7eaa.png)